contracts/0.4.24/oracle/LidoOracle.sol

```diff
@@ -86,6 +86,9 @@ contract LidoOracle is ILidoOracle, AragonApp {
     bytes32 internal constant LAST_COMPLETED_EPOCH_ID_POSITION = keccak256("lido.LidoOracle.lastCompletedEpochId");
     bytes32 internal constant PREV_COMPLETED_EPOCH_ID_POSITION = keccak256("lido.LidoOracle.prevCompletedEpochId");
 
+    /// @dev function credentials to be called when the quorum is reached
+    bytes32 internal constant QUORUM_DELEGATE_POSITION = keccak256("lido.LidoOracle.quorumDelegate");
```

Comment:
"Delegate call" has a specific meaning in eth, let's not use this word

Reply:
Renaming to `QUORUM_CALLBACK`...